.github/workflows/algolia.yml

```diff
@@ -24,12 +24,24 @@ jobs:
     - run: cd website && npm ci
     - name: Run delete script
       run: cd website && npm run delete-index
-    - name: Registry login
+    - name: Algolia Indexing
       run: |
         docker run --rm -e "CONFIG=$(cat ./website/config.json | jq -r tostring)" \
           -e API_KEY=${{secrets.ALGOLIA_UPDATE_API_KEY}} \
           -e APPLICATION_ID=${{secrets.ALGOLIA_APPLICATION_ID}} \
           algolia/docsearch-scraper
+
+    - name: Slack Notification
+      if: always()
+      uses: ravsamhq/notify-slack-action@v1
```

Comment:
Does this action give us more functionality that the verified [slack-send](https://github.com/marketplace/actions/slack-send) action? This is not a blocker for the PR to be merged

Reply:
[slack-send](https://github.com/marketplace/actions/slack-send) action needs a lot of tweaks to get a properly formatted message. 

The action I'm using in this PR is specifically for reporting GH workflow status and requires less customization